File: 1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        f={}
        for char in s:
            if char in f:
                f[char]+=1
            else:
                f[char]=1
        for char in t:
            if char in f:
                f[char]-=1
            else:
                f[char]=-1
        change = 0
        for cnt in f.values():
            change += abs(cnt)
        return change//2

File: 1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        unordered_map <char, int> hashmap;
        for (char c:s) {
            hashmap[c]++;
        }
        for (char ch:t) {
            hashmap[ch]--;
        }

        int count=0;
        for (auto it:hashmap) {
            count+=abs(it.second);
        }
        return count/2;
    }
};